assets/scripts/ui/__tests__/RangeSlider.test.js

```diff
@@ -35,17 +35,17 @@ describe('RangeSlider', () => {
       // we are able to use the `.toHaveBeenCalledWith()` method below.
       event.persist()
     })
-    const { getByLabelText } = render(
-      <RangeSlider onChange={handleChange}>foo</RangeSlider>
-    )
-    const input = getByLabelText('foo')
+    render(<RangeSlider onChange={handleChange}>foo</RangeSlider>)
+    const input = screen.getByLabelText('foo')
 
     // Initial value is expected to be halfway between the default
     // mininum and maximum range, and per HTML spec, the value is
     // of type string
     expect(input.value).toBe('50')
 
     // Change the value
+    //  TODO: user-event library dosen't seem to have a way to do this
+    // or it isn't documented, check back later
```

Comment:
I think you are right on this. The library doesn't currently seem to support non-text input elements. (There's a similar issue where they don't support `<input type="time" />`.